SparkPost/2web2ui
# FE- 968 - Restyling of Billing Pages

FE- 968 - Restyling of Billing Pages

### What Changed
 - Updated all Pages of Billing

### How To Test
Mocks: https://sparkpost.invisionapp.com/public/share/J6144MQ7CN#/screens
 - Check billing Page for a manual customer
 - Check billing for user with pending plan change
 - Check immediate plan change page (Downgrade to free)
 - try upgrading plan and check select plan page.

*Notes* - rebased of off FE-927
### To Do
- [x] use `PanelSectionTableCollection` once FE-963 is deployed
